dsi/plugins/tests/test_file_consumer.py

```diff
@@ -65,6 +65,18 @@ def test_csv_plugin_adds_rows_multiple_files():
     assert len(plug.output_collector.keys()) == 15
 
 
+def test_csv_plugin_adds_rows_multiple_files_strict_mode():
```

Comment:
Does this test fail if `plug.add_rows()` does not throw a `TypeError`? [PyTest `raises`](https://docs.pytest.org/en/stable/reference/reference.html#pytest.raises) may be useful.